packages/eas-cli/src/new/templates/readme-additions.md

```diff
@@ -1,3 +1,20 @@
+## Get started
```

Comment:
This is mostly copied from the pre-existing readme, but that gives you a numbered list that's 1) Install dependencies 2) run start. We've already installed deps, so I'm getting rid of that.